CMakeLists.txt

```diff
@@ -297,6 +322,44 @@ if(LINUX OR BSD)
         src/detection/media/media_linux.c
         src/detection/wmtheme/wmtheme_linux.c
         src/detection/font/font_linux.c
+        src/detection/qt.c
+        src/detection/gtk.c
+    )
+endif()
+
+if(WIN_MSYS)
+    list(APPEND LIBFASTFETCH_SRC
+        src/common/processing_linux.c
+        src/detection/host/host_windows.cpp
+        src/detection/bios/bios_windows.cpp
+        src/detection/board/board_windows.cpp
+        src/detection/cpu/cpu_windows.cpp
+        src/detection/gpu/gpu_windows.cpp
+        src/detection/battery/battery_windows.cpp
+        src/detection/displayserver/displayserver_windows.c
+        src/detection/wmtheme/wmtheme_windows.c
+        src/detection/opengl/opengl_windows.c
+        src/detection/users/users_windows.cpp
+        src/detection/os/os_windows.cpp
+        src/detection/processes/processes_windows.cpp
+        src/detection/disk/disk_windows.c
+        src/detection/cpuUsage/cpuUsage_windows.c
+        src/detection/cpuUsage/cpuUsage_nowait_windows.cpp
+        src/detection/memory/memory_windows.cpp
+        src/detection/font/font_windows.cpp
+        src/detection/terminalshell/terminalshell_linux.c
+        src/detection/terminalshell/terminalshell_windows.cpp
+        src/detection/packages/packages_linux.c
+        src/detection/packages/packages_windows.c
+        src/detection/kernel/kernel_windows.cpp
+        src/detection/localip/localip_windows.c
+        src/util/windows/wmi.cpp
+
+        # Shared
+        src/detection/terminalfont/terminalfont_linux.c
+        src/detection/poweradapter/poweradapter_linux.c
+        src/detection/media/media_linux.c
+        src/detection/temps/temps_linux.c
```

Comment:
I would prefer to have "Not implemented" methods for those, since all of them won't work on windows.